src/terminal.rs

```diff
@@ -165,6 +165,13 @@ impl DefaultTerminal {
         mortal::Terminal::new().map(DefaultTerminal)
     }
 
+    #[cfg(any(unix))]
```

Comment:
I don't think it's a good idea to simply add this as an inherent method with a `#[cfg(unix)]` attribute. This could allow a user to use this method without being aware that it is a Unix-only extension. The typical Rust convention in this situation is to use an extension trait, as mortal does with the `OpenTerminalExt` trait.

Reply:
Agreed, Let’s drop this PR